deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -90,6 +90,9 @@ master:
     # The name of the service account to use.
     # If not set and create is true, a name is generated using the fullname template
     name:
+  
+  # specify how many old ReplicaSets for the Deployment to retain.
+  revisionHistoryLimit: 3
```

Comment:
I'd be inclined to use Kubernetes defaults (i.e. leave this empty). Or then have good argumentation why certain value (in this case 3) was picked. WDYT?

Reply:
Well the issue asked for setting the value to 3, but I think the default in Kubernetes is 10. I'm good with both :)

Reply:
Ah, yes. @omerap12 just leave the fields empty (in values.yaml) and we should be good

Reply:
you got it @marquiz